emission/core/wrapper/suggestion_sys.py

```diff
@@ -309,22 +309,24 @@ def review_start_loc_nominatim(lat, lon):
 def calculate_yelp_server_suggestion_singletrip_nominatim(uuid, tripid):
     user_id = uuid
     timeseries = esta.TimeSeries.get_time_series(user_id)
-    cleaned_trips = timeseries.get_data_df("analysis/cleaned_trip", time_query = None)
+    cleaned_trips = timeseries.get_entry_from_id("analysis/cleaned_trip", tripid)
+    # timeseries.get_data_df("analysis/cleaned_trip", time_query = None)
     '''
-    I changed this to be directly indexing the cleaned trips, wanted to ask 
-    to make sure that it wasn't reading everything to memory
+    Used the abstract time series method, wanted to make sure this was what you were asking for
     '''
     
     # spec_trip = cleaned_trips.iloc[cleaned_trips[cleaned_trips._id == tripid].index.tolist()[0]]
-    spec_trip = cleaned_trips[cleaned_trips._id == tripid]
-    start_location = spec_trip.start_loc.tolist()[0]
-    end_location = spec_trip.end_loc.tolist()[0]
+    # cleaned_trips[cleaned_trips._id == tripid]
+    # spec_trip.end_loc.tolist()[0]
+    spec_trip = cleaned_trips.get("data")
+    start_location = spec_trip["start_loc"]
+    end_location = spec_trip["end_loc"]
```

Comment:
this is an entry object, so you can directly access `cleaned_trips.data.start_loc`